### PR TITLE
Feat: Formatted License Page

### PR DESCRIPTION
**Pull request by theswerd**
### Formatted license page
This PR makes the license page match the style of the rest of the app.

#### Screenshots
Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-30 at 01 01 11
Android:
Screenshot_1588233321

#### How did you test the change?
* [x] iOS Simulator
* [x] Android Emulator
* [ ] iOS Device
* [ ] Android Device
* [ ] `curl` to a dev App Engine server

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [ ] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

File: client/flutter/lib/pages/license_page.dart [hunk 1]
```diff
@@ -1,481 +1,25 @@
+// BASED OFF OF FLUTTER'S about.dart file
 // Copyright 2014 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// Originally from:
```
Comment (advayDev1) [inline client/flutter/lib/pages/license_page.dart:5]: Put this comment back.

File: client/flutter/lib/pages/license_page.dart [hunk 2]
```diff
@@ -1,481 +1,28 @@
...
-  /// [IconTheme]'s [IconThemeData.size].
-  final Widget applicationIcon;
-
-  /// A string to show in small print.
-  ///
-  /// Typically this is a copyright notice.
-  ///
-  /// Defaults to the empty string.
-  final String applicationLegalese;
+class WHOLicensePage extends StatefulWidget {
```
Comment (advayDev1) [inline client/flutter/lib/pages/license_page.dart:18]: No need for WHO prefix.

File: client/flutter/lib/pages/license_page.dart [hunk 3]
```diff
@@ -552,80 +93,44 @@ class _LicensePageState extends State<LicensePage> {
...
-            bottom: false,
-            child: Scrollbar(
-              child: ListView(
-                padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 12.0),
-                children: <Widget>[
-                  Text(name, style: Theme.of(context).textTheme.headline5, textAlign: TextAlign.center),
-                  if (icon != null) IconTheme(data: Theme.of(context).iconTheme, child: icon),
-                  Text(version, style: Theme.of(context).textTheme.bodyText2, textAlign: TextAlign.center),
+    return Localizations.override(
+      locale: const Locale('en', 'US'),
```
Comment (advayDev1) [inline client/flutter/lib/pages/license_page.dart:104]: What's this about?

File: client/flutter/lib/pages/license_page.dart [hunk 4]
```diff
@@ -1,481 +1,28 @@
...
+class WHOLicensePage extends StatefulWidget {
+  const WHOLicensePage();
 
   @override
-  _LicensePageState createState() => _LicensePageState();
+  _WHOLicensePageState createState() => _WHOLicensePageState();
 }
 
-class _LicensePageState extends State<LicensePage> {
+class _WHOLicensePageState extends State<WHOLicensePage> {
```
Comment (advayDev1) [inline client/flutter/lib/pages/license_page.dart:25]: Remove WHO prefix everywhere

File: client/flutter/lib/pages/license_page.dart (hunk 2 above)
Comment (theswerd) [reply client/flutter/lib/pages/license_page.dart:18]: LicensePage already exists in the material library so it wouldn't work while material was imported.

Fixed by renaming to LicensePage and removing the unnecessary material import.

File: client/flutter/lib/pages/license_page.dart (hunk 3 above)
Comment (theswerd) [reply client/flutter/lib/pages/license_page.dart:104]: This was in the original, I assumed it was because the licenses were only available in english.

**Pull request merged**